app/scripts/run-dependencies.sh

```diff
@@ -13,6 +13,22 @@ docker run --rm \
     $registry/montagu-cert-tool:$cert_tool_version \
     gen-keypair /workspace
 
+# prepare to create orderly db
+rm $PWD/demo -rf
+rm $PWD/git -rf
+mkdir $PWD/demo
+ORDERLY_IMAGE="vimc/orderly:master"
+OW_MIGRATE_IMAGE="vimc/orderlyweb-migrate:master"
+OW_CLI_IMAGE="vimc/orderly-web-user-cli:master"
+
+#create orderly db
+docker pull $ORDERLY_IMAGE
+docker run --rm --entrypoint create_orderly_demo.sh -v "$PWD:/orderly" -u $UID -w /orderly $ORDERLY_IMAGE .
```

Comment:
This is a docker in docker issue - volume mounts here relate to locations on the host machine because the docker sock is mounted from the host. Basically this apporach won't work, I'll have a little think about best alternatives...